resources/views/components/pagination.blade.php

```diff
@@ -0,0 +1,54 @@
+@if($paginator->hasPages())
+    <nav class="flex justify-center items-center mt-5 gap-x-1">
+        <a @if(!$paginator->onFirstPage()) href="{{ $paginator->previousPageUrl() }}" @endif>
+            <button type="button"
+                    class="min-h-[38px] min-w-[38px] py-2 px-2.5 inline-flex justify-center items-center gap-x-1.5 text-sm rounded-lg text-gray-800 hover:bg-gray-100 focus:outline-none focus:bg-gray-100 disabled:opacity-50 disabled:pointer-events-none dark:text-white dark:hover:bg-white/10 dark:focus:bg-white/10"
+                    @if($paginator->onFirstPage()) disabled @endif>
+                <svg class="flex-shrink-0 size-3.5" xmlns="http://www.w3.org/2000/svg" width="24" height="24"
+                     viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
+                     stroke-linejoin="round">
+                    <path d="m15 18-6-6 6-6"/>
+                </svg>
+                <span>
+                    {{ __('common.previous') }}
+                </span>
+            </button>
+        </a>
+        <div class="flex items-center gap-x-1">
+            @php
+                // Calculate the start and end of the range (5) of pages to show.
```

Comment:
Ik zou zelf de range vergroten naar 10, zelf heb ik veel lege ruimte op de pagina (zou ook aan mijn resolutie kunnen liggen)

Reply:
Die 5 zegt eigenlijk alleen dat er tussen Vorige en Volgende 5 linkjes staan. Hoeveel accounts er op 1 pagina staan is bepaald in de controller met `->paginate(x)`, die heb ik op 6 gezet, maar kan ik wel verhogen!